app/views/funds/show.html.erb

```diff
@@ -128,18 +136,23 @@
     </div>
 
     <div class="col-xl-9">
+      <h2>Funded by</h2>
```

Comment:
Should this check for the existence of any funders?

Reply:
There should always be at least one funder, otherwise it wouldn't have run an allocation